### PR TITLE
[IMP] database danagement: added db deletion instructions

### PR DESCRIPTION
**Pull request by dbo-odoo**
@tivisse @mart-e 
Can I haz merge plz?

### CODE REVIEW

File: db_management/db_online.rst [hunk 1]
```diff
@@ -7,45 +7,75 @@
...
 
+To manage your databases, access the `database management page <https://www.odoo.com/my/databases>`__
+(you will have to sign in). Then click on the
+`Manage Your Databases <https://www.odoo.com/my/databases/manage>`__ button.
+
+.. image:: media/databases.png
+    :align: center
+
+Make sure you are connected as the Administrator of the database you
+want to manage - many operations depends on indentifying you remotely to that database.
```
Comment (mart-e) [inline db_management/db_online.rst]: 80 char limit per page you shall respect (and all the occurrences below)

File: db_management/db_online.rst [hunk 2]
```diff
@@ -7,45 +7,75 @@
...
+
+Several actions are available:
+
+.. image:: media/db_buttons.png
```
Comment (mart-e) [inline db_management/db_online.rst]: through [pngquant](https://pngquant.org/), your images you shall compress

File: db_management/db_online.rst [hunk 3]
```diff
@@ -7,45 +7,75 @@
...
+* Contact Support
+    Access our `support page <https://www.odoo.com/help>`__
+    with the correct database already selected
+
+
 .. _duplicate_online:
 
+Deleting a database
+===================
+
```
Comment (mart-e) [inline db_management/db_online.rst]: empty section you shall not add

File: db_management/db_online.rst [hunk 4]
```diff
@@ -7,45 +7,75 @@
...
+    Upgrade your database to the latest Odoo version to enjoy cutting-edge features
+* :ref:`Duplicate <duplicate_online>`
+    Make an exact copy of your database, if you want
+    to try out new apps or new flows without compromising
+    your daily operations
+* Rename
+    Rename your database (and its URL)
+* **Backup**
+    Download an instant backup of your database; note that we
+    already do that for you according to our Online SLA
```
Comment (mart-e) [inline db_management/db_online.rst]: that we already download or already backup, probably rephrase

File: db_management/db_online.rst [hunk 5]
```diff
@@ -55,3 +85,36 @@ database.
 
     .. image:: media/dup_expires.png
         :align: center
+
+.. _delete_online_database:
+
+Deleting a Database
+===================
+
+You can only delete databases of which you are the Administrator.
```
Comment (mart-e) [inline db_management/db_online.rst]: `administrator` lowercase, he has already too much power, don't make him feel more important than he is ;-)

File: db_management/db_online.rst [hunk 6]
```diff
@@ -55,3 +85,36 @@ database.
...
+
+.. danger:: When you delete your database all the data will be permanently lost.
+    The deletion is instant and for all the Users. We strongly suggest to backup
```
Comment (mart-e) [inline db_management/db_online.rst]: one more empty line to create a real line return in rst.
`users` lowercase.
I thought Odoo was making backup. What happens to them?
"suggest **you** to backup" ?

File: db_management/db_online.rst (hunk 1 above)
Comment (dbo-odoo) [reply db_management/db_online.rst]: Well shit, I told @efaOdoo the limit was 100 -_-

File: db_management/db_online.rst (hunk 5 above)
Comment (dbo-odoo) [reply db_management/db_online.rst]: You mean `ADMINISTRATOR`?

**Comment by dbo-odoo** [comment]
@mart-e 
:up:

**Comment by tivisse** [comment]
Done at https://github.com/odoo/documentation-user/commit/a0b5401c4d802f56b0660b866efe2b3cbc6d4d9c

Thanks for your work.

**Comment by dbo-odoo** [comment]
Thx @tivisse for working late :)

**Pull request merged**